dev/TeachingTip/TeachingTip.cpp

```diff
@@ -923,8 +926,8 @@ void TeachingTip::OnShouldConstrainToRootBoundsChanged()
     // If we have opened the tip's popup and then this property changes we will need to discard the old popup
     // and replace it with a new popup.  This variable indicates this state.
 
-    //The underlying popup api is only available on 19h1 plus, if we aren't on that no opt.
-    if (SharedHelpers::Is19H1OrHigher())
+    // If the popup isn't created or doesn't have the ShouldConstrainToRootBounds api then no opt
+    if(m_popup.get().try_as<winrt::Controls::Primitives::IPopup3>())
```

Comment:
```suggestion
    if (m_popup.get().try_as<winrt::Controls::Primitives::IPopup3>())
```

Reply:
Isn't this what is already there?

---
In reply to: [270511566](https://github.com/Microsoft/microsoft-ui-xaml/pull/504#discussion_r270511566) [](ancestors = 270511566)